### ISSUE TITLE
Prisma Cloud rule update

### ISSUE THREAD

**Issue by hk-dassana** [dassana-io/dassana#328]
We want to make sure that the JQ rules we are writing for **`filters`** and **`risk-config`** do not give error when a JSON key is not found. For example-
currently we have these for prisma cloud-
```
.resource.rrn | contains("rrn")
.message.resource.rrn | contains("rrn") 
```
both of these rules are giving error when we send a JSON not having prisma cloud alerts. For example, this JSON-
```
{
"foo":"bar"
}
```
gives error
```
jq: error (at <stdin>:2): null (null) and string ("rrn") cannot have their containment checked exit status 5
```
when such error occurs, engine considers the error as rule not matching and logs the error.
in other words, we need to first check for presence of keys and then you perform contains check.

**Comment by hk-dassana** [comment]
code rolled out on 10/04/21
